assets/syntaxes/Packages/LaTeX/syntax_test_latex.tex

```diff
@@ -124,7 +124,7 @@ \subsubsection{name}
 %           ^ support.function
 %
 
-\parencite[Propositon~1]{Ref}
+\parencite[Proposition~1]{Ref}
 % ^ meta.function.citation.latex
 % ^ support.function.cite.latex
 %          ^ meta.group.bracket.latex
```

Comment:
This is confusing, the `^` for `meta.group.bracket.latex` points to after the `[`.
I wonder if the `^` for `constant.other.citation.latex` should point to the `e` in `Ref`.